apps/main/src/store/createGlobalSlice.ts

```diff
@@ -38,21 +37,17 @@ type GlobalState = {
   isXLgUp: boolean
   layoutHeight: LayoutHeight
   loaded: boolean
-  locale: Locale['value']
   pageWidthPx: number | null
   pageWidth: PageWidthClassName | null
-  maxSlippage: { [key: string]: string }
   routerProps: RouterProps | null
   showScrollButton: boolean
-  themeType: Theme
 }
 
 // prettier-ignore
 export interface GlobalSlice extends GlobalState {
```

Comment:
Unrelated, but this is pretty concerning: https://github.com/curvefi/curve-frontend/blob/9f0db9e97443d20d7eab9f01afb99679d61db1ce/apps/main/src/store/useStore.ts#L1

Reply:
Huh, I removed it and prettier only applied to one line... Was introduced near the initial commit at the beginning of 2024; probably a leftover debugging thing?